kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeSharePanel.js

```diff
@@ -767,32 +725,29 @@ define([
             const orgData = [];
             const noMatchedOrgFoundStr = 'Search by Organization name';
 
-            $.fn.select2.amd.require(
-                ['select2/data/array', 'select2/utils'],
-                (ArrayData, Utils) => {
-                    if (!orgList) return;
-                    orgList.forEach((org) => {
-                        orgData.push({ id: org.id, text: org.name });
-                    });
-                    $inputOrg.select2({
-                        formatNoMatches: noMatchedOrgFoundStr,
-                        placeholder: function () {
-                            return $(this).data('placeholder');
-                        },
-                        delay: 250,
-                        width: '40%',
-                        data: orgData,
-                        minimumResultsForSearch: 0,
-                        allowClear: true,
-                        language: {
-                            noResults: function () {
-                                return noMatchedOrgFoundStr;
-                            },
+            $.fn.select2.amd.require(['select2/data/array', 'select2/utils'], () => {
```

Comment:
whitespace changes, and `ArrayData` and `Utils` aren't used. But I guess they need to be loaded.